lib.php

```diff
@@ -2135,7 +2137,7 @@ function facetoface_user_cancel($session, $userid=false, $forcecancel=false, &$e
  * @returns string Error message (or empty string if successful)
  */
 function facetoface_send_notice($postsubject, $posttext, $posttextmgrheading,
-                                $notificationtype, $facetoface, $session, $userid) {
+                                $notificationtype, $facetoface, $session, $userid,$htmlmessage) {
```

Comment:
looks like you're missing a space after the comma - I'm sure this code doesn't comply well, but we shouldn't be adding new issues ;-)